### DISCUSSION TITLE
part-ii/2017-140/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#265] [Question comments]
Part II 2017 Statistical Physics: Paper 3, Section II, D 

https://questions.tripos.org/part-ii/2017-140/

**Comment by thenewtonium** [comment]
<details><summary> Part (a) (technically bookwork but this confused me)</summary>

Heat is *absorbed* in the isothermal expansion, and *emitted* during isothermal compression.

Work is done *on* the gas during all compression steps, and *by* the gas during all expansion steps.
</details>

<details><summary> Part (b) </summary>

(i) Most useful form to write the internal energy is as $E = \frac{3}{2}pV$

(ii) Have $TdS = dE + pdV = \frac{5}{2}p dV + \frac{3}{2} V dp$. 

The heat *supplied* along $AB$ is $\int_A ^B T dS = \frac{3}{2}p_0 V_0$. ($TdS \geq 0$ along $AB$).

The heat *emitted* along $CA$ is $-\int_C ^A T dS = \frac{5}{2}p_0 V_0$. ($TdS \leq 0$ along $CA$)

(iii) Along $BC$ you should get that $TdS = p_0(\frac{15}{2} -4\frac{V}{V_0})dV$. Going along $BC$ we have $dV >0$, so along $BC$ heat is *supplied* to the gas for $V < \frac{15}{8}V_0$ and *emitted* for $V > \frac{15}{8}V_0$.

(iv) To calculate $Q$ we can use (iii) to calculate the heat supplied along $BC$, by integrating $T dS$ along the segment of $BC$ with $V < \frac{15}{8}V_0$, then add this to the heat supplied along $AB$ to get $Q = \frac{97}{32}$[^first].

To calculate $W$, there's a neat trick: if you remember Green's theorem from vec. calc. (or think about it a bit), for a closed contour $C$, we have that $W = \oint_C p dV$ is just (up to a sign) the area inside $C$ in the $(p,V)$-plane. So the total work done is just the area of the triangle $ABC$, i.e. $\frac{1}{2} p_0 V_0$.

Hence $\eta = \frac{16}{97}$[^first].

[^first]: At least, that's what I got, and iirc (since I didn't write it down) it's what my supervisor said the answer was.
